### PR TITLE
Improve performance of data initialization

### PR DESCRIPTION
**Pull request by yrodiere**
`RelationshipSizeIT` is down from 25 minutes to 5 minutes on my machine. Still not great, but that's the difference between "kind of usable" and "no way I'll even try".

There are probably other things to optimize, I just stopped because I spent too much time on this.

### CODE REVIEW

File: model/src/main/java/org/hibernate/performance/search/model/service/AnswerFiller.java [hunk 1]
```diff
@@ -40,44 +42,39 @@ public AnswerFiller(SessionFactory sessionFactory) {
...
-				int index = quotient + i;
-
-				T answer = elements.get( index );
-				action.accept( answer );
-				session.merge( answer );
 			}
 		} );
 	}
 
+	public <T> void batch(Session session, ScrollableResults elements, Class<T> elementType, Consumer<T> action) {
```
Comment (fax4ever) [inline model/src/main/java/org/hibernate/performance/search/model/service/AnswerFiller.java:68]: unused?

**Review by fax4ever — COMMENTED** [review]
I have one comment + there is a test not working on CI.
I tried to explore the cause, but I haven't found anything...

File: model/src/main/java/org/hibernate/performance/search/model/service/AnswerFiller.java (hunk 1 above)
Comment (yrodiere) [reply model/src/main/java/org/hibernate/performance/search/model/service/AnswerFiller.java]: Right, fixed.

**Comment by yrodiere** [comment]
Thanks, I force-pushed an update. Let's see if the build passes; if so, I'd say the failure was caused by a docker container that wasn't removed properly in a previous (failing) build.

**Pull request merged**